Keas.Mvc/ClientApp/components/Equipment/EquipmentContainer.tsx

```diff
@@ -0,0 +1,100 @@
+import PropTypes from "prop-types";
+import * as React from "react";
+
+import { AppContext, IEquipment, IEquipmentAttribute, IPerson } from "../../Types";
+
+import AssignEquipment from "./AssignEquipment";
+import EquipmentList from "./EquipmentList";
+
+interface IState {
+    loading: boolean;
+    equipment: IEquipment[];
+}
+
+export default class EquipmentContainer extends React.Component<{}, IState> {
+  public static contextTypes = {
+    fetch: PropTypes.func,
+    person: PropTypes.object,
+    team: PropTypes.object
+  };
+  public context: AppContext;
+  constructor(props) {
+    super(props);
+
+    this.state = {
+        equipment: [],
+      loading: true
+    };
+  }
+  public async componentDidMount() {
+    // are we getting the person's equipment or the team's?
+    const equipmentFetchUrl = this.context.person
+      ? `/equipment/listassigned/${this.context.person.id}`
```

Comment:
@laholstege we've been forgetting but remember we'll have to put the team name in the URL so the controller knows what team we are getting the equipment for (not strictly needed for listassigned but needed for list, and good for both.